js/reducers/hendelser.js

```diff
@@ -7,42 +8,16 @@ const initiellState = {
     hentet: false,
 };
 
-const hendelser = (state = initiellState, action = {}) => {
-    switch (action.type) {
-        case actiontyper.HENT_HENDELSER_FEILET: {
-            return {
-                ...state,
-                hentingFeilet: true,
-                henter: false,
-                hentet: true,
-            };
-        }
-        case actiontyper.HENTER_HENDELSER: {
-            return {
-                ...state,
-                henter: true,
-                hentingFeilet: false,
-                hentet: false,
-            };
-        }
-        case actiontyper.HENDELSER_HENTET: {
-            return {
-                ...state,
-                henter: false,
-                hentingFeilet: false,
-                data: action.hendelser.map((h) => {
-                    return {
-                        ...h,
-                        inntruffetdato: new Date(h.inntruffetdato),
-                    };
-                }),
-                hentet: true,
-            };
-        }
-        default: {
-            return state;
-        }
-    }
-};
+const hendelser = createReducer(
+    actiontyper.HENT_HENDELSER_FEILET,
+    actiontyper.HENTER_HENDELSER,
+    actiontyper.HENDELSER_HENTET,
+    initiellState,
+    (h) => {
```

Comment:
Pet peeve: Kan du skrive ut hele navnet på variablen i stedet for å bruke en bokstav? ;)